lib/membrane_audio_mixer_bin.ex

```diff
@@ -20,55 +20,28 @@ defmodule Membrane.AudioMixerBin do
 
   require Membrane.Logger
 
+  alias __MODULE__.MixerOptions
```

Comment:
modules defined inside the other module are aliased by default, this alias is not needed